book/pipeline-service.md

```diff
@@ -1,28 +1,50 @@
 # Pipeline Service
 
-Pipeline Service provides a SaaS for pipelines. It leverages:
+Pipeline Service provides Tekton APIs and services to RHTAP.
+In the initial phase of RHTAP, Pipeline Service will be provided by a stock
+installation of the OpenShift Pipelines operator.
+This deployed version will be the a candidate build of the OpenShift Pipelines
+operator from a Red Hat build system.
 
-- OpenShift for the compute.
-- Tekton Pipelines, Chains, Pipelines as Code, Results for the core of the service.
-- OpenShift GitOps / Argo CD for managing the infrastructure.
+![Pipelines operator deployment](../diagrams/pipeline-service.drawio.svg)
 
-The following features are provided:
+## APIs and Services
 
-- Signing and attestation of `TaskRuns` with [Tekton Chains](https://tekton.dev/docs/chains/).
-- Archiving/Pruning of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/) (not activated).
-- Integrations with GitHub, Gitlab, and Bitbucket with [Pipelines as Code](https://pipelinesascode.com) (PaC).
+Pipeline Service provides the following:
+
+- Tekton APIs directly through its custom resource definitions.
+- Container image signing and provenance attestations through Tekton Chains.
+- Archiving of `PipelineRuns`, `TaskRuns`, and associated logs through Tekton
+  Results.
+
+Pipeline Service also exposes the following ingress points:
+
+- Pipelines as Code Webhook: this is a `Route` that receives webhook events
+  from source code repositories.
+- Tekton Results REST: this is an `Ingress` that serves Tekton Results data
+  over a RESTful API. Clients authenticate with the same `Bearer` token used to
+  authenticate Kubernetes requests.
```

Comment:
I feel like we should highlight the exporter and that we are providing additional metrics outside of what we currently find upstream, with also noting we are also submitting metrics directly upstream, as well as considering migrating our more complicated downstream metrics upstream once they have matured and solidified.